fabric-events-interaction-v0/src/testmod/java/net/fabricmc/fabric/test/event/interaction/PlayerBreakAndPlaceBlockTests.java

```diff
@@ -21,10 +21,11 @@
 
 import net.minecraft.block.Blocks;
 
+import net.fabricmc.fabric.api.event.player.PlayerBlockPlaceEvents;
 import net.fabricmc.api.ModInitializer;
 import net.fabricmc.fabric.api.event.player.PlayerBlockBreakEvents;
 
-public class PlayerBreakBlockTests implements ModInitializer {
+public class PlayerBreakAndPlaceBlockTests implements ModInitializer {
```

Comment:
I'd much rather see a seperate entrypoint for placing tests

Reply:
Are we able to have two ModInitializer classes loaded at the same time?